packages/data-stores/src/site/types.ts

```diff
@@ -101,7 +101,6 @@ export interface SiteDetails {
 	launch_status: string;
 	jetpack: boolean;
 	is_fse_eligible: boolean;
```

Comment:
Let's remove `is_fse_eligible`, as well, since that's also deprecated. I think it would be helpful to add a comment about both these properties being deprecated, so it's clear that they're not just missing after we remove them.